mailbox/backup/src/test/java/org/apache/james/mailbox/backup/DefaultMailboxBackupTest.java

```diff
@@ -0,0 +1,225 @@
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.mailbox.backup;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+
+import org.apache.commons.compress.archivers.zip.ZipFile;
+import org.apache.james.blob.api.BlobId;
+import org.apache.james.blob.api.BlobStore;
+import org.apache.james.blob.api.HashBlobId;
+import org.apache.james.blob.memory.MemoryBlobStore;
+import org.apache.james.core.User;
+import org.apache.james.junit.TemporaryFolderExtension;
+import org.apache.james.mailbox.MailboxManager;
+import org.apache.james.mailbox.MailboxSession;
+import org.apache.james.mailbox.MessageManager;
+import org.apache.james.mailbox.backup.ZipAssert.EntryChecks;
+import org.apache.james.mailbox.backup.zip.ZipArchivesLoader;
+import org.apache.james.mailbox.backup.zip.Zipper;
+import org.apache.james.mailbox.extension.PreDeletionHook;
+import org.apache.james.mailbox.inmemory.MemoryMailboxManagerProvider;
+import org.apache.james.mailbox.model.MailboxId;
+import org.apache.james.mailbox.model.MailboxPath;
+import org.apache.james.mailbox.store.mail.model.Mailbox;
+import org.apache.james.mailbox.store.mail.model.MailboxMessage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.reactivestreams.Publisher;
+import org.testcontainers.shaded.org.apache.commons.io.IOUtils;
+
+import com.github.fge.lambdas.Throwing;
+import reactor.core.publisher.Mono;
+
+@ExtendWith(TemporaryFolderExtension.class)
+class DefaultMailboxBackupTest implements MailboxMessageFixture {
+
+    private static final String USER = "user";
```

Comment:
A good point to make tests more readable is to give some real name to the users. Consider the simpsons, or other cartoons.

A bit of fun makes it less boring!

Reply:
I agree :)  but it's hard coded in some existing mailboxpath of test. I have started refactoring to reuse existing constants. The real takeaway from your point is that the usernames should be moved to MailboxMessageFixture.

Reply:
Yes. Let's get this right for newly written code, at least.

Distinct commits for older tests classes are cool if not too dificult to do imo...